app/repositories/mandataris.js

```diff
@@ -0,0 +1,34 @@
+import { API, STATUS_CODE } from 'frontend-lmb/utils/constants';
+
+export default class MandatarisRepository {
+  async isActive(mandatarisId) {
+    const response = await fetch(
+      `${API.MANDATARIS_SERVICE}/mandatarissen/${mandatarisId}/isActive`
+    );
+    const jsonReponse = await response.json();
+
+    if (response.status !== STATUS_CODE.OK) {
+      console.error(jsonReponse.message);
+      throw jsonReponse.message;
+    }
+
+    return jsonReponse.isActive;
+  }
+
+  async getBestuursperiode(mandatarisId) {
```

Comment:
I think it is not worth doing something like this, we only use this in the fractie repository and it gives us a headache to actually retrieve the model again, so just move this logic to the mandataris-service, when we want to update the currentFractie, we give a mandataris, which is linked to a bestuursperiode, so you don't need to pass this.